bbc/simorgh
# [WSTEAM1-362] Fix Related Content Section on Amp (for Naidheachdan pages)

Resolves JIRA [WSTEAM1-362]

Overall change
======
Fixes bug with RelatedContentSection on Amp. Currently, the RelatedContentSection does not render correctly on Amp if  images are not supplied.
(note: this bug is only seen on naidheachdan amp pages)

Code changes
======

- Add styling for `RelatedContentSection` on Amp
- Add styling for `RelatedContentGrid` and `PromoItem` on Amp
- Add logic to apply styling on `naidheachdan` Amp pages
- Add logic to remove image placeholder if there is no image

Testing
======
1. Checkout this PR
2. Go to http://localhost:7080/naidheachdan/sgeulachdan/c18q7nedn2ko.amp and observe the Related Content section at the bottom of the page

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
